docs/client-side/yeoman.rst

```diff
@@ -0,0 +1,260 @@
+
+Yeoman is *opinionated*, that is, it prescribes tools and best practices for your target technologies so you don't have to decide "Use version X or Y?" or "What directory structure should I use?" Yeoman gets you stared with a known-good project that runs.
+
+The Yeoman command line tool yo works alongside a Yeoman generator. Generators define the technologies that go into a project. Here are a few sample `generators <http://yeoman.io/generators/>`_:
+
+- `AngularJS Generator <https://github.com/yeoman/generator-angular>`_ creates a starting point for a new single-page Angular application. 
+- `JQuery Generator <https://github.com/yeoman/generator-jquery>`_ creates the shell code for a jQuery plug-in.
+- `Chrome App Generator <https://github.com/yeoman/generator-chromeapp>`_ generates everything you need to create an extension for the Chrome browser.
+- The `Flux-React generator <https://github.com/banderson/generator-flux-react>`_ creates an application based on Facebook's Flux/React architecture.
+- `ASPNET generator <https://github.com/OmniSharp/generator-aspnet>`_ builds several types of ASP.NET 5 applications including Empty, Console, Web, Web API, Nancy ASP.NET, and Class Library.
+- You can even `generate your own Yeoman generator <https://github.com/yeoman/generator-generator>`_.
+
+In this article:
+	- `Getting Started`_
+	- `Integrating Generated Projects with Visual Studio`_
+	- `Generating a Webapp`_
+	- `Generating a Chrome Plugin`_
+
+Getting Started
+---------------
+Let's start by installing Yeoman without relying on the Visual Studio installation. 
+
+1.	Download and install `Node.js <https://nodejs.org/>`_ from https://nodejs.org. Leave the defaults to install the npm package manager and to make Node and npm accessible globally.
+
+.. image:: yeoman/_static/yeoman-nodejs-setup.png
+ 
+2.	Open a command line such as Windows PowerShell, and run the command to list all NPM packages found globally. The list will also show you where the packages are located on your system.
+
+.. code-block:: console
+
+ npm ls -g
+  
+3.	Run the command to install the Gulp command line interpreter:
+
+.. code-block:: console
+
+ npm install -g gulp-cli
+
+4.	On the command line, install the ASP.NET generator:  
+
+.. code-block:: console
+
+ npm install -g generator-aspnet
+
+.. note:: The **–g** flag installs the generator globally so that you can use it from any path on your system. The **yo** command and supporting Node.js modules are installed automatically.
+ 
+5.	Make a new directory where your project will be generated:
+
+.. code-block:: console
+
+ mkdir c:\MyYo
+
+6.	On the command line, make the new directory the current directory.
+
+.. code-block:: console
+ 
+ cd c:\MyYo
+
+7.	Run the ``yo`` command and pass the name of the generator.
+
+.. code-block:: console 
+
+ yo aspnet
+
+8.	The aspnet generator displays a menu. Leave the default **Empty Application** type selected and press ``Enter``.
```

Comment:
Instead of Empty Application let's suggest they pick the desired application type